plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/delete/DeleteFilter.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg.delete;
+
+import io.trino.plugin.iceberg.IcebergColumnHandle;
+import org.apache.iceberg.Schema;
+
+import java.util.List;
+
+public interface DeleteFilter
+{
+    Schema schema();
+
+    RowPredicate createPredicate(List<IcebergColumnHandle> columns);
```

Comment:
what's the meaning of columns parameter passed here?

Reply:
These are the columns in the pages that will be passed to the filter.